web3j/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java

```diff
@@ -46,10 +50,21 @@
   private final SecureTokenApi api;
   private final CountDownLatch countDownLatch;
   private final MetricsService metrics;
+  private final Web3jConfig web3jConfig;
 
+  @SneakyThrows
```

Comment:
Sneaky throws is not appropriate for run() method. We need to capture and record token transfer failures too.